### ISSUE TITLE
Fork a project in DCS before importing a project from someone else's repo

### ISSUE THREAD

**Issue by benjore** [unfoldingWord/translationCore#5992]
### User Story

As a team member working on a group project, I want tC to fork the original project to my repo in DCS so that when I'm done and upload the project to my repo, I can simply issue a PR to merge it back into the "master" branch.

### Features / Specifications

- [ ] When a user is logged in and imports a project from someone else's repo:
  - [ ] Fork the project to my repo (and then import it from there??)
  - [ ] Upload to the forked project (in my repo)
- [ ] When a user is NOT logged in and imports a project from someone else's repo: 
  - [ ] ~Capture the necessary data to allow for a fork to happen later~
  - [ ] ~If the user logs in and attempts to upload the project to their repo, then using the data captured during the import - fork the project to the user's repo and then upload.~
  - [ ] No forking will happen

### Definition of Done

- [ ]
- [ ]
- [ ]

**Comment by jag3773** [comment]
Note that the forking tree information is arguably more important than needing to do a PR later, but we get both with a fork :)

**Comment by benjore** [comment]
Koz - need to make it an option
Manny - merge conflicts? 
Spidel - what about name changes?

**Comment by PhotoNomad0** [comment]
@jag3773  - since the new project has same history, couldn’t we do merge requests now?  Or is it that DCS doesn’t give that option?

**Comment by jag3773** [comment]
Yeah you could do that manually.  The problem is that we are not preserving the trail of data.

Recording the fork in DCS provides many advantages for workflow, debugging, and general provenance.

Koz - I don't think it should be an option, it should just be what tC does, just like if you were to copy a repo on DCS, you hit the fork button, that's just the way it works.  If you really want to subvert the system you can download it yourself and push it back up to your account but the default behavior should be to preserve the relationship.

Manny - this shouldn't pose any merge conflict issues.

Spidel - this is unrelated to any naming changes or issues.

To be clear, I'm just talking about hitting this API endpoint: https://git.door43.org/api/swagger#/repository/createFork .  Should just be a single request and then the rest of the import process would be identical to what (I expect) you have now.
